infrastructure/terraform/modules/activation/main.tf

```diff
@@ -74,14 +74,35 @@ module "bigquery" {
   default_table_expiration_ms = 360000000
 }
 
+resource "null_resource" "check_artifactregistry_api" {
+  provisioner "local-exec" {
+    command = <<-EOT
+    while ! gcloud services list | grep -i "artifactregistry.googleapis.com"
```

Comment:
We need to have some timeout in case there are issues. Perhaps 5 minutes is a good option.